recipes/jaeger/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jaeger" %}
+{% set version = "1.14.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{name }}-{{ version }}-darwin-amd64.tar.gz  # [osx]
```

Comment:
Is there a reason that this cannot be compiled from source?  We generally do not repackage binaries for compatability reasons

Reply:
No reason besides laziness.

It's all statically linked, I am curious what the compatibility issues would be.

I copied the structure from this go package which also just downloads the binaries: https://github.com/conda-forge/go-spatial-feedstock/blob/master/recipe/meta.yaml

Are there best practices for distributing go binaries in conda forge?

Reply:
Oh if this is a go library, I am not sure what the best practices are.  CC @conda-forge/help-go!

Reply:
So for golang stuff we tend to just build it.  See our terraform / terragrunt feedstocks

Reply:
One of the advantages of building with Go is that it's really easy to cross compile binaries. Is there a way in conda forge to say "We just need to run this one CI job on linux that will produce binaries for linux, mac, and windows"

Reply:
So my go-spatial feedstock was created when Go was not available as a conda package so building it here was not an option back then.